dlp/src/test/java/dlp/snippets/QuickstartTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc.
+ * Copyright 2019 Google LLC
```

Comment:
We don't update copyright dates

Reply:
This is actually a new file - Git just thinks it was moved because it was similar to a deleted file.